### ISSUE TITLE
[Feature Request] Add option to change package name and app name

### ISSUE THREAD

**Issue by MateChan** [LSPosed/LSPatch#179]
### Is your feature request related to a problem?/你的请求是否与某个问题相关？

There is no problem during use, but it is a request for additional functions that would be nice to have.

### Describe the solution you'd like/描述你想要的解决方案

Option to change package name and app name of target app

### Additional context/其他信息

I thought it would be useful to be able to install different versions of the same app to check the compatibility of each version of the module and target app.

For example, by installing a patch for each of the versions of the Twitter app that we use on a daily basis and the latest version in parallel, we can check the compatibility of the latest version and the module while using Twitter on a daily basis.

Thank you for your consideration.

**Comment by yujincheng08** [comment]
too difficult
